hypertrace-gradle-publish-plugin/README.md

```diff
@@ -17,30 +18,13 @@ all of which, with the exception of license, can be omitted if left unchanged.
 ```kotlin
  hypertracePublish {
       license // REQUIRED to be a value defined in org.hypertrace.gradle.publishing.License
-      vcsUrl // Optional. Defaults to the env var CIRCLE_REPOSITORY_URL
-      user // Optional. Defaults to gradle property publishUser
-      apiKey // Optional. Defaults to gradle property publishApiKey
-      repo // Optional. Defaults to "maven"
-      organization  // Optional. Defaults to "hypertrace"
-      name // Optional. Publication name, defaults to project.getName()
-      apiUrl // Optional. Defaults to https://api.bintray.com
+      url  // Optional.
```

Comment:
this is the pom url right? A few different thoughts:
- If we keep it, we should rename it for clarity
- If we expect it to be set, we should make it required
- If we're really just trying to provide an avenue to setting it, the whole pom is already accessible in the usual fashion to any consumers, since we're just preconfiguring the vanilla publishing mechanism now - so I think my inclination is to just remove it (the difference with license is there we both want to require it to be set and limit the available options for HT consumers)

Reply:
1. Renamed it to `pomUrl`.
2. Added default value.